connector_clickup/models/project/common.py

```diff
@@ -0,0 +1,199 @@
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+from odoo.addons.component.core import Component
+
+
+class ClickupProjectProject(models.Model):
+    _name = "clickup.project.project"
+    _inherit = ["clickup.binding"]
+    _inherits = {"project.project": "odoo_id"}
+    _description = "Clickup project.project binding model"
+
+    odoo_id = fields.Many2one("project.project", required=True, ondelete="restrict")
+    synced_at = fields.Datetime(readonly=True)
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+    _description = "Inherited project.project model"
+
+    clickup_bind_ids = fields.One2many(
+        "clickup.project.project",
+        "odoo_id",
+        readonly=True,
+    )
+    external_id = fields.Char(
+        related="clickup_bind_ids.external_id",
+        readonly=True,
+    )
+    clickup_backend_id = fields.Many2one(
+        "clickup.backend",
+        related="clickup_bind_ids.backend_id",
```

Comment:
The user will manually set this while exporting,no need to add backend ids with the related param.

Reply:
Removed store = True, as per the discussion